models/public/index.md

```diff
@@ -223,6 +223,14 @@ The task of speech recognition is to recognize and translate spoken language int
 | ----------------- | -------------- | ------------------------------------------------------------ | -------- | ------ | ------- |
 | DeepSpeech V0.6.1 | TensorFlow\*   | [mozilla-deepspeech-0.6.1](./mozilla-deepspeech-0.6.1/mozilla-deepspeech-0.6.1.md) | 7.55%    | 0.0472 | 47.2    |
 
+## Image Translation
+
+The task of image translation is to generate the output based on exemplar.
+
+| Model Name | Implementation | OMZ Model Name                     | Accuracy | GFlops | mParams |
+| -----------| -------------- | ---------------------------------- | -------- | ------ | ------- |
+| CoCosNet   | PyTorch\*      | [cocosnet](./cocosnet/cocosnet.md) | 12.98dB  |        |         |
```

Comment:
please look at other composite models as an example and make it similar in index file